Tests/LibWeb/Ref/reference/svg-background-no-natural-size-ref.html

```diff
@@ -0,0 +1,14 @@
+<style>
+  * {
+    margin: 0;
+  }
+  body {
+    background-color: white;
+  }
+</style>
+<!-- To rebase:
+  1. Open svg-background-no-natural-size.html in Ladybird
+  2. Right click > "Take Full Screenshot"
+  3. Update the image below:
+-->
+<img src="./images/svg-background-no-natural-size-ref.png">
```

Comment:
Isn't Ref for ref tests, while this is basically a pixel test?

Reply:
There's a bunch of tests like this already (I don't think there's a separate place for pixel tests)

Reply:
Right, we don't have a dedicated mechanism for pixel tests atm.